css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
+    color: #eeeeee;
+}
+
+/* Navigation */
+
+.navbar {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.nav-link,
+.navbar-brand {
+    color: #dedede !important;
+    cursor: pointer !important;
+}
+
+.nav-link {
+    margin-right: 1em !important;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: #af0404 !important;
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: rgb(255, 0, 0, 0.7) !important;
```

Comment:
Why use `!important`? It's a poor practice that does more harm than good. Usually `!important` should be avoided unless you have a very good reason to use it.

Reply:
https://www.freecodecamp.org/news/learn-bootstrap-4-in-30-minute-by-building-a-landing-page-website-guide-for-beginners-f64e03833f33/#navbar
I was following this tutorial.
I was thinking this is used to override the bootstrap properties

For example if I remove !important from 
.navbar-brand {
  color: #dedede ;
  cursor: pointer;
}
than the text color of my navigator goes back to balck

The background colors work though without !important

Reply:
I could change it to navbar-dark to change color, but now its white, still not sure how to customize it without using !importanr

Reply:
To approach, you need to create a selector with hight specificity than Bootstrap. If you add `.c-navbar` to the element that has `.navbar-brand`, it should work:

```css
/* bootstrap css */
.navbar-dark .navbar-brand {
  color: #fff;
}
```

```css
/* your css  */
.navbar-dark .c-navbar.navbar-brand {
  color: #dedede;
}
```

Now the CSS specificity is higher. Here's [a CSS specificity explanation](https://dev.to/emmabostian/css-specificity-1kca). This [game](https://flukeout.github.io/) is my favorite to learn it. For example, the answer to the first level is `plate`. And here's an [online specificity calculator](https://polypane.app/css-specificity-calculator/). This is one of the most important things about you need to know about CSS to build organized styles.